circleguard/visualizer.py

```diff
@@ -654,22 +685,37 @@ def increase_speed(self):
 class VisualizerWindow(QMainWindow):
     def __init__(self, replays=[], beatmap_id=None, beatmap_path=None):
         super(VisualizerWindow, self).__init__()
+        self.is_fullscreen = False
+        self.setAutoFillBackground(True)
         self.setWindowTitle("Visualizer")
         self.setWindowIcon(QIcon(str(resource_path("resources/logo.ico"))))
         self.interface = _Interface(replays, beatmap_id, beatmap_path)
         self.setCentralWidget(self.interface)
-        self.setWindowFlag(Qt.MSWindowsFixedSizeDialogHint) # resizing is not important rn
         QShortcut(QKeySequence(Qt.Key_Space), self, self.interface.pause)
         QShortcut(QKeySequence(Qt.Key_Right), self, lambda: self.interface.change_frame(reverse=False))
         QShortcut(QKeySequence(Qt.Key_Left), self, lambda: self.interface.change_frame(reverse=True))
         QShortcut(QKeySequence(Qt.Key_Up), self, self.interface.increase_speed)
         QShortcut(QKeySequence(Qt.Key_Down), self, self.interface.lower_speed)
         QShortcut(QKeySequence(Qt.CTRL + Qt.Key_F11), self, self.toggle_frametime)
+        QShortcut(QKeySequence.FullScreen, self, self.go_fullscreen)
+        QShortcut(QKeySequence(Qt.Key_F), self, self.go_fullscreen)
+        QShortcut(QKeySequence(Qt.ALT + Qt.Key_Return), self, self.go_fullscreen)
```

Comment:
alt+return? is this a standard keybinding for fullscreen?

Reply:
osu! uses this as as the fullscreen toggle
See https://osu.ppy.sh/help/wiki/Shortcut_key_reference